src/Nethermind/Nethermind.Evm/Precompiles/PointEvaluationPrecompile.cs

```diff
@@ -38,8 +38,12 @@ static bool IsValid(in ReadOnlyMemory<byte> inputData)
 
             ReadOnlySpan<byte> inputDataSpan = inputData.Span;
             ReadOnlySpan<byte> versionedHash = inputDataSpan[..32];
-            ReadOnlySpan<byte> z = inputDataSpan[32..64];
-            ReadOnlySpan<byte> y = inputDataSpan[64..96];
+            Span<byte> z = stackalloc byte[32];
```

Comment:
maybe add SkipLocalsInit attribute? There are lot of long locals

Reply:
done